make/config.mk

```diff
@@ -37,8 +37,8 @@
 # choice of compiler
 #--------------------
 
-export CC = gcc
-export CXX = g++
+#export CC = gcc
```

Comment:
It would be better to set it if not set already

Reply:
I don't like it, the norm is to use the environment setting like in 99% of other projects.

Reply:
What if someone does not set them explicitly? I agree with Anton that we should provide a default configuration instead of relying on the people having to set environment variables just to get compilation working

Reply:
gcc is used as default, again as how the rest of the world does it. I don't think is a good idea to set this on config.mk . is standard to pick it up from the $CC env variable. You don't need to set the env variable explicitly as gcc is taken by default. Do you have evidence that this is not the case?